.gitignore

```diff
@@ -38,11 +38,17 @@ __pycache__
 # image files
 *.png
 
+/docs/site/*
```

Comment:
Should there be a leading `/` here? I think that makes it an absolute path.

Reply:
`/` makes it directly under the root folder, i.e., only {WORKSPACE}/docs/site will be ignored and {WORKSPACE}/folder1/docs/site won't be ignored.